modules/sandboxed-containers-query-metrics.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-querying-metrics_{context}"]
+= Viewing metrics for {sandboxed-containers-first}
+
+You can access the metrics for {sandboxed-containers-first} in the Metrics UI in the web console.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed.
+* You have {sandboxed-containers-first} installed.
+* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects.
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Observe* → *Metrics*.
+
+. In the input field, enter the query for the metric you want to observe. You can type *kata* to view a list of all kata-related metrics.
```

Comment:
Since autocompletion is enabled, starting typing kata will list all the kata metrics available, since they all start with "kata_".
So, I would change with something like:
"Starting typing 'kata' in the input form will trigger autocompletion and will list all the kata available metrics, as all of them start with 'kata_'.